api/models.py

```diff
@@ -1,3 +1,25 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
 
-# Create your models here.
+ACTIVITY_LEVEL = (
+			("SC", "Sleepy_Cat"),
+			("LC", "Lazy_Cat"),
+			("AC", "Active_Cat"),
+			("MC", "Mentor_Cat"),
+	)
+
+class UserProfile(models.Model):
+	user = models.OneToOneField(User) 
+	#id = models.AutoField(primary_key=True) 
+	fullname = models.CharField(max_length=100, blank=False)
+	githubuser = models.CharField(max_length=100, null=False, blank=False)
+	email = models.EmailField(blank=False)
```

Comment:
There is a User model already defined in Django by default, The UserProfile Model is meant to compliment the data that we will need, but the User model will not capture, so email field is actually part of the User model.

I would encourage to utilize the default model as much as possible and let the UserProfile model to play the "support" role in user authentication process

Reference: https://docs.djangoproject.com/en/1.7/topics/auth/default/